tutorials/restic-s3-backup/index.mdx

```diff
@@ -0,0 +1,232 @@
+- You have [installed and configured the AWS-CLI](/storage/object/api-cli/object-storage-aws-cli/) to use with Scaleway Object Storage
+- You have [created an Object Storage bucket](/storage/object/how-to/create-a-bucket/)
+
+</Message>
+
+## Installing Restic
+
+1. Connect to your Instance with SSH. In a terminal run:
+
+  ```
+  ssh root@<your_instance_ip>
+  ```
+
+2. Update the system and upgrade the software already installed on the Instance.
+
+  ```
+  apt update && apt upgrade -y
+  ``` 
+
+3. Install Restic. 
+
+  ```
+  apt-get install restic
+  ```
+
+4. Ensure the installation was successful by checking the version.
+
+  ```
+  restic version
+  ```
+
+## Setting up the S3 repository
+
+A repository is the storage space where your backups will be hosted. In this tutorial, we will use Scaleway Object Storage buckets to host our backups.
+
+1. Set up your [environment variables](https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#environment-variables), if you have not done so yet. Replace `$SCW_ACCESS_KEY`, `$SCW_SECRET_KEY` and `$SCW_DEFAULT_PROJECT_ID` by their corresponding values.
+
+  ```
+  export SCW_ACCESS_KEY=$SCW_ACCESS_KEY
+  export SCW_SECRET_KEY=$SCW_SECRET_KEY
+  export SCW_DEFAULT_PROJECT_ID=$SCW_DEFAULT_PROJECT_ID
+  ```
+
+  Restic requires permission to perform the `PutObject` action on the bucket. In the next step we will implement a [bucket policy](/storage/object/concepts/#bucket-policy) to grant the permissions. 
+
+2. Create a file named `bucket-policy.json` using a text editor of your choice. In this tutorial we use `nano`.
+
+  ```
+  nano bucket-policy.json
+  ``` 
+
+  Paste the following content:
+
+  ```json
+  {
+    "Version": "2012-10-17",
+    "Id": "MyBucketPolicy",
+    "Statement": [{
+      "Sid": "access for restic",
+      "Effect": "Allow",
+      "Principal": {
+        "SCW": "project_id:<SCW_PROJECT_ID>"
+      },
+      "Action": [
+        "s3:PutObject",
+        "s3:ListBucket",
+        "s3:GetObject",
+        "s3:GetObject",
+        "s3:DeleteObject"
+      ],
+      "Resource": [
+        "<BUCKET_NAME>",
+        "<BUCKET_NAME>/*"
+      ]
+    }]
+  }
+  ``` 
+
+  <Message type="note">
+
+  In this example we include other actions beyond `PutObject`. You can configure your policies according to your preferences. To learn more, refer to the [reference documentation](/storage/object/api-cli/bucket-policy/).
+
+  </Message>
+
+3. Run the following command to apply the policy:
+
+  ```
+  aws s3api put-bucket-policy --bucket <bucket_name> --policy file://bucket-policy.json
+  ```
+
+  If the operation is successful, no output is returned.
+
+4. Initialize the repository using your bucket's endpoint as a backend. Your bucket endpoint can be found in the **Bucket Settings** tab under **Bucket Information**. 
+
+  ```
+  restic -r s3:https://s3.fr-par.scw.cloud/<bucket_name> init
+  ```
```

Comment:
Aren't we missing the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to use S3? cf https://restic.readthedocs.io/en/latest/080_examples.html#initializing-the-restic-repository